Documentation/artifact-format.md

```diff
@@ -332,6 +544,20 @@ For more information about the script and state API, see the official Mender
 documentation.
 
 
+header-augment.tar.gz (version 3)
+-------------
+
+Format: tar
+
+This file is complementing the information contained in the header.tar.gz.
+It can have the same structure as header.tar.gz, but for security reasons
+(this file is not signed) only certain files and parameters are allowed.
+
+At the moment ONLY header-info and type-info files are allowed which can
+contain only `artifact_provides`, `artifact_depends` `rootfs_image_checksum`
+parameters and type of the update.
```

Comment:
I think only `artifact_depends` or? Because the end result is supposed to be identical, so `artifact_provides` can not be different.

Reply:
Right!